### PR TITLE
feat(component): implement fullwidth variant for expressive model

### PR DESCRIPTION
**Pull request by sannminwin**
### Related Ticket(s)

#808 Create fullwidth variant for Expressive Model Component

### Description

This pull request implements fullwidth variant for expressive model

expressive-model-fullwidth

### Design Specs
[box folder](https://ibm.ent.box.com/file/573186081225)

### Changelog

**New**

- added fullwidth variant

### Deploy Preview
https://deploy-preview-817--ibmdotcom-react-experimental.netlify.com/?path=/story/expressive-modal--fullwidth

### CODE REVIEW

File: packages/react/src/components/ExpressiveModal/ExpressiveModal.js [hunk 1]
```diff
@@ -20,13 +20,21 @@ const { prefix } = settings;
  * Expressive Modal
  *
  * @param {object} props react proptypes
+ * @param {boolean} props.fullwidth sets whether the modal is fulldiwth or not
```
Comment (jeffchew) [inline packages/react/src/components/ExpressiveModal/ExpressiveModal.js]: @sannminwin can you also add this new prop to the README?

Comment (sannminwin) [reply packages/react/src/components/ExpressiveModal/ExpressiveModal.js]: updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/817/commits/d93508d6d14583824606a25a36c762dbe1f5cee4)

File: packages/react/src/components/ExpressiveModal/README.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | ----------- | -------- | --------------- | ------------- | --------------------------------------------------- |
 | `children`  | NO       | Node            | null          | Components/Elements to be rendered within the Modal |
 | `className` | NO       | String OR Array | null          | Custom classname(s) for the Modal                   |
+| `fullwidth` | NO       | Boolean         | flase         | Sets whether the Modal is fullwidth or not          |
```
Comment (jeffchew) [inline packages/react/src/components/ExpressiveModal/README.md]: @sannminwin `false` is misspelled...

Comment (sannminwin) [reply packages/react/src/components/ExpressiveModal/README.md]: that's too bad, I am updating now.

**Pull request merged**